src/pages/Registration/Steps/Dashboard/index.tsx

```diff
@@ -55,7 +55,7 @@ function Dashboard() {
       <Step
         num={2}
         disabled={isStepDisabled}
-        status={`Level ${documentation.level}`}
```

Comment:
[EndowmentTierNum](https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/RC-v1.8/src/types/contracts/common.ts#L55), [TDocumentation.Tier](https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/RC-v1.8/src/types/aws/ap/registration.ts#L70), [FilterState.tiers](https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/RC-v1.8/src/slices/components/marketFilter/types.ts#L20), `endow_tier` in AWS - it just a good coding practice to have all these field names aligned as much as possible.

Reply:
I personally have to constantly remind myself `level` is the same thing as `tier` when I'm looking at this code.

Reply:
Hopefully you can agree that it's important to have all team members comfortable working with **our** code, especially when it's something we can all agree is good practice.

Reply:
> EndowmentTierNum, TDocumentation.Tier, FilterState.tiers, endow_tier in AWS - it just a good coding practice to have all related field/type names aligned as much as possible.

If this in general true then we won't even have intermediary types and just use server types directly on our forms/ui but we name/structure things as they are closely used to the UI we are working on

> I personally have to constantly remind myself level is the same thing as tier when I'm looking at this code.

personally this code looks consistent to me
`Level ${documentation.level}`

why is level being mapped as tier in this UI ? 
`Level ${documentation.tier}`

Reply:
> but we name/structure things as they are closely used to the UI we are working on

You won't be looking at UI that much, but you will be looking at its internal structure. 
If the UI says "Level" but all the other internal & backend types/fields say "tier", it is better to keep things consistent **internally**.

Reply:
We should follow the contract and/or AWS DB naming vs. how the UI refers to things for best consistency. The UI naming/label could change tomorrow (Level 1 >> Green; Level 2 >> Yellow; etc and month later we decide to move to labeling them as Tier 1; Tier 2; etc). The point being that we wouldn't want to change the code attribute with each UI change, since the DB and contracts both refer to tier concept. It just helps unify that a bit.